src/components/BounceRulesActivity/RuleFilter/index.jsx

```diff
@@ -0,0 +1,85 @@
+import React from "react";
+import "./index.scss";
+import { TextInput } from "@sendgrid/ui-components/text-input";
+import { Select } from "@sendgrid/ui-components/select";
+import { Row } from "../../Row";
+import { Column } from "../../Column";
+
+const RuleFilter = ({
+  searchToken,
+  updateSearchToken,
+  updateSearchCategory,
+  filterOptions,
+  addFilter,
+  removeFilter,
+  invalidFilter,
+}) => (
+  <div className="filter-wrap">
+    <div className="filter-header">
+      <p className="filter-title">
+        Search emails by{" "}
+        <a className="filter-option-toggle" href="#">
+          all
+        </a>{" "}
+        of the following:
+      </p>
+      <div className="btn-list">
+        <button type="button" className="btn btn-small btn-secondary">
+          Clear Search
+        </button>
+      </div>
+    </div>
+    <div className="filter-list">
+      <Row>
+        <Column width={3} offset={1}>
+          <div style={{ paddingTop: "15px" }} />
```

Comment:
Let's put this inline style in a class and possibly wrap the div around the Select component so it's not an empty div spacer if it's possible to wrap it